apps/stress-test/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
it would be a great follow-up if we could have following setup

https://github.com/microsoft/fluentui/pull/23807/files#diff-563dd8ceff51e02df6c807890c6fdb64648a8b6295b321a82cefd41183c544d0

Reply:
Added a follow up: #24425